packages/theme/src/components/indicator.ts

```diff
@@ -3,6 +3,7 @@ import { isGray, shadeColor } from "@yamada-ui/utils"
 
 export const Indicator: ComponentStyle = {
   baseStyle: ({ withBorder }) => ({
+    position: "absolute",
```

Comment:
It seems to me that lines 7-15 need to be migrated as well.

Reply:
It seems to me that lines 7-15 need to be migrated as well and I think you need to add the deleted one to theme.